lib/utils/default.ts

```diff
@@ -0,0 +1,18 @@
+import * as t from "io-ts";
+
+/**
+ * Sets properties default values when calling t.validate() method on models
+ * see https://github.com/gcanti/io-ts/issues/8
+ */
+// tslint:disable:no-any
+export function withDefault<T extends t.Any>(
+  type: T,
+  defaultValue: t.TypeOf<T>
+): t.Type<any, t.TypeOf<T>> {
+  return new t.Type(
+    type.name,
+    (v: any): v is T => type.is(v),
+    (v: any, c: any) => type.validate(v !== undefined ? v : defaultValue, c),
```

Comment:
`v !== undefined && v !== null` ?

Reply:
probably it's better to not overwrite a property explicitely set to null, ie:

```
{
  foobar: null
}
```

Reply:
probably it's better to not overwrite any property explicitely set to null

Reply:
not sure about that, everywhere in the code there's no type for which `null` is a valid value, meaning that if `v === null`, validation fails